drivers/entropy/Kconfig.mcux

```diff
@@ -32,7 +32,7 @@ config ENTROPY_MCUX_RNG
 
 config ENTROPY_MCUX_CAAM
 	bool "MCUX CAAM driver"
-	default y
+	default n
```

Comment:
`default n` is redundant, simply removing `default y` should be enough.

also if this driver is disabled due to an issue, it would be nice to comment out `default y` instead of removing it, and add a comment on what the issue is.